topostats/grains.py

```diff
@@ -76,10 +78,14 @@ def __init__(
         self.threshold_std_dev = threshold_std_dev
         self.threshold_absolute_lower = threshold_absolute_lower
         self.threshold_absolute_upper = threshold_absolute_upper
+        self.absolute_area_threshold_dict = {
```

Comment:
Would it be easier/simpler to make the configuration file provide a dictionary?

Something like

```yaml
  absolute_area_thresholds:
    upper: [500, 800] # above surface [Low, High] in nm^2 (also takes null)
    lower: [null, null] # above surface [Low, High] in nm^2 (also takes null)
```

Would mean you can pass in `absolute_area_threshold` as a dictionary directly.

```python
        self.absolute_area_threshold = absolute_area_threshold     
```

Reply:
Yes I like this idea a lot! I'll make this change in a sec.